content/articles/email-parser-python/index.md

```diff
@@ -0,0 +1,170 @@
+### Parsing emails in Python
+
+Perhaps you are a machine learning engineer trying to build an email spam classifier, and you want to get some way of pre-processing the emails, or you are trying to look for some correlations between some emails, you will in one way or another have to parse the emails first. We will look at how to do that using Python. Python has the `email` module containing methods to help us achieve this.
+
+#### Prerequisites
+
+The following are the pre-requisites for this article
+An understanding of Python.
+Jupyter notebook installed or access to Google Colab.
+
+> We will use Jupyter notebooks in this article. You may use regular Python files for the code executions.
+
+A Colab notebook with the codes is found [here](https://colab.research.google.com/drive/1nDz58G4cDukqukOlPRVST-jKBK0AyMwa?usp=sharing).
+
+#### Getting started
+
+We will parse emails from the SpamAssassin website then look at their structure and contents. The site contains spam and normal emails. We will try and look at their structures using the metrics highlighted. 
+
+First, create a notebook called **email_parser.ipynb** or your preferred name.
+
+Then, create a cell and add the packages we need.
```

Comment:
Don't mention steps that involve creating a cell. You may rephrase it as... In a new cell, import the required packages.